### ISSUE TITLE
Dependency Warnings

### ISSUE THREAD

**Issue by mickelsonm** [peterruggero/postmaster-go#8]
I am getting dependency warnings when using this library:

`2015/05/11 10:04:20 WARNING: Package "github.com/jmcvetta/restclient" is deprecated.`
`2015/05/11 10:04:20 WARNING: It has been replaced by "github.com/jmcvetta/napping".`
`2015/05/11 10:04:20 WARNING: All bugfixes and new development will occur in Napping.`

Is there a plan to update the library to use these new dependencies? Right now I am on the fence about whether I should use this library in my project or if I should roll my own. Definitely would like to use the official library if I can though!

Thanks!

**Comment by tlianza** [comment]
We're using this client, but I'm also a bit concerned about the warnings. Did you wind up rolling your own? I probably won't until we run into a tangible issue.

**Comment by mickelsonm** [comment]
Nope, I didn't roll my own. I basically just forked the postmaster library, then I baked the deprecated restclient library into it. I know it's hacky/dirty, but I didn't have time to refactor the library to work with the mentioned `napping` library like it suggests. Perhaps in the future someone could update it....not a mission critical thing, but the warning messages I found annoying in my application.

[Here](https://github.com/aries-auto/postmaster-go) is that forked version if you're curious to see it. 

Good luck!
